doc/dev/design/workflows/storage_devices_and_osds.rst

```diff
@@ -0,0 +1,304 @@
+==============================================
+Storage devices and OSD’s management workflows
```

Comment:
Nit:
```suggestion
Storage Devices and OSDs Management Workflows
```

Reply:
I would make "OSD" singular in this phrasal adjective (OSD Management) or complex noun (OSD Management Workflows).

I suggest:

Storage Devices and OSD-Management Workflows

Reply:
Another possibility that reduces ambiguity is "OSD-Management Workflows", making "OSD-Management" explicitly and unambiguously a complex adjective.